meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -5,7 +5,7 @@ set -e
 DATA_PARTITION=/dev/disk/by-label/resin-data
 
 echo -n "Expand ext4 partition on $DATA_PARTITION... "
-e2fsck -y -f $DATA_PARTITION
+e2fsck -y $DATA_PARTITION
```

Comment:
It was different, but it got closed as Andrei pointed out that he would like fsck at every boot.